README.md

```diff
@@ -44,6 +40,16 @@ To install CVPy, use the following command:
 
 `pip install sas-cvpy`
 
+### Install and Run Jupyter Notebook
```

Comment:
After running these steps (on Windows 11), I still couldn't launch a notebook. I got the error: `Jupyter command jupyter-notebook not found.` Cvpy and mayavi work fine. 

I'm content with adding a command to install the notebook app or pointing cvpy users to the [Jupyter install instructions](https://jupyter.org/install).

Reply:
Hey Kyle, thanks for testing those out. I updated the instructions again, can you please let me know if those work for you? Otherwise, I'll link users to the instructions you provided.

Reply:
Yep, that works! Now that I'm in Jupyter, that brings up another point in the ipykernel line.